packages/content/README.md

```diff
@@ -0,0 +1,148 @@
+- _literal metadata_ like `title`
+- _inferred metadata_ such as `coverage` (global, regional, local) derived from the geographic extent, 
+- _linked metadata_ `license` created through `licenseInfo`
+- _associated resources_ such as datasets via `operationalLayers` or `owner` (`as IHubMember`) via the `item.owner`
+
+### Methods
+
+- `getContent(id, associatedOptions, hubRequestOptions)`
+- `createContent(id, attributes, hubRequestOptions)`
+- `updateContent(id, attributes, hubRequestOptions)`
+- `deleteContent(id, hubRequestOptions)`
+
+`associatedOptions` is proposed to support external references that may require additional XHR to fetch. Similar to JSON-API or GraphQL. 
+For example `associatedOptions: { itemInfo: true, itemData: true, connectedContent: 5 }`
+
+- Question: should download formats + links be optional associations or require XHR via the download components?
+
+### Metadata
+
+Based on the [Hub Content UI Inventory](https://app.lucidchart.com/invitations/accept/dab82a25-2ed9-4f63-8075-3db8873087e3)
+
+Attemping a simple _Hungarian Notation_ for attributes suffix: `*Date`, `*Id`, `*Number`. Plural (e.g. `tags`) is an array.
+
+```ts
+export interface IContentModel extends IModel {
+    id: string; 
+    type: 'document' | 'dataset' | 'map' | 'app' | 'site' | 'initiative' | 'event' | 'template' ;
+    owner: IHubMember;
+    
+    access: {
+        visibility: 'private' | 'org' | 'public';
+        permission?: 'view' | 'edit' | 'admin'; // based on User requesting the info
+        updateGroups?: Array<IGroup>;
+        viewGroups?: Array<IGroup>;
+    };
+
+    metrics?: {
+        visibility: 'private' | 'org' | 'updateGroups' | 'public'  ; // TODO: should we share with specific groups?
+        viewNumber?: number; // available depending on visibility; 
+    };
+
+    // Common metadata
+    title: string;
+    summary: string;
+    description: string;
+    license: IHubLicense; // title, description, and optional link to license item with more info
+    source: IHubOrg | IHubTeam | IHubMember; // each of these has common metadata like `title`, `thumbnail` , and `link`
+    createdDate: Date; 
+    publishedDate: Date;
+    updatedDate: Date;
+    geography: IHubGeography; // {title, coverage (global|regional|local), extent (WSEN), geometry, source}
+    categories?: Array<string>;
+    tags: Array<string>;
+    language: string; // written locale e.g. en, es, cn
+    contentUrl: string; // Link the the raw content
+    thumbnailUrl: string; // Full URL
+    status?: string; // e.g. [in-development, maintained, archived, unreviewed, prototype]
+
+    // Unique or additional formal metadata that will be displayed in sidebar
+    metadata?: {
+        // Additional custom/formal metadata elements
+    };
+
+    // Content specific attributes such as 
+    attributes?: {
```

Comment:
not sure about using `attributes` for this mainly out of concern for potential confusion w/ `dataset.attributes`. So this will be `dataset.attributes` stripped of the above properties?

Reply:
it would be good to clarify w/ comments below which of these exist already